src/patternfly/components/CalendarMonth/calendar-month.scss

```diff
@@ -80,16 +80,6 @@
 .pf-c-calendar-month__header-nav-control {
   margin-right: var(--pf-c-calendar-month__header-nav-control--MarginRight);
   margin-left: var(--pf-c-calendar-month__header-nav-control--MarginLeft);
-
-  &.pf-m-prev-month {
-    --pf-c-calendar-month__header-nav-control--MarginRight: var(--pf-c-calendar-month__header-nav-control--m-prev-month--MarginRight);
```

Comment:
we actually still need these - it's part of preserving the CSS var API - if we remove the use of them in the code, they no longer work if a user has defined them with a custom margin. That's effectively why we preserve the variables<sup>*</sup>, so they can be used.

<sup>*</sup> I think a better approach would be to remove the variable declaration we've marked for removal, and just use "0" (or whatever the variable has been set to) as a fallback, and mark this for removal. We could also do the same with `--pf-c-calendar-month__header-nav-control--Margin[Left/Right]` since those are no longer necessary, either. That would save let us remove some unnecessary code. Though that's a separate issue, we could do it here - wdyt @mattnolting @srambach?

Reply:
I like the falback solution @mcoker 
If there isn't one already, should we agree on a standard comment adjacent so that we can easily find them to remove?

Reply:
@mcoker I think that, in general, this is a better approach than setting vars marked for deprecation to `0`. @thatblindgeye Not a blocker, but feel free to make that update!